packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/CryptoSelection/CryptoSelector/index.tsx

```diff
@@ -258,11 +258,12 @@ const CryptoSelector: React.FC<InjectedFormProps<{}, Props> &
               coinOrder.map(coin => {
                 const accounts = props.accounts[coin] as Array<SwapAccountType>
                 return accounts.map(
-                  account =>
+                  (account, index) =>
                     account.balance !== '0' &&
                     account.balance !== 0 &&
                     isInvitedShowNC(account) && (
                       <CryptoAccountOption
+                        key={index}
```

Comment:
Can you use something like `account.id`? That's better

Reply:
not sure do we have it but will check